src/ert/gui/ertwidgets/create_experiment_dialog.py

```diff
@@ -36,16 +38,15 @@ def __init__(
 
         experiment_label = QLabel("Experiment name:")
         self._experiment_edit = StringBox(
-            TextModel(""), placeholder_text="My experiment", minimum_width=200
-        )
-        self._experiment_edit.setValidator(
-            NotInStorage(notifier.storage, "experiments")
+            TextModel(""), placeholder_text="My_experiment", minimum_width=200
         )
+        self._experiment_edit.setValidator(ExperimentValidation(notifier.storage))
 
         ensemble_label = QLabel("Ensemble name:")
-        self._ensemble_edit = QLineEdit()
-        self._ensemble_edit.setMinimumWidth(200)
-        self._ensemble_edit.setPlaceholderText("My ensemble")
+        self._ensemble_edit = StringBox(
+            TextModel(""), placeholder_text="My_ensemble", minimum_width=200
```

Comment:
Not really related to this PR, but should perhaps make a better placeholder text 😆